GroundDBMemoryPool/src/rdma/rdma_client/connect.cc

```diff
@@ -31,10 +31,17 @@ struct resources *connectServer(
         fprintf(stderr, "failed to connect QPs\n");
         return NULL;
     }
+    if (poll_completion(res))
+    {
+        fprintf(stderr, "poll completion failed\n");
+        return NULL;
+    }
+    // Verify connection by matching the string VERIFIER sent by the server
+    if (strcmp(res->buf, VERIFIER))
+    {
+        fprintf(stderr, "failed to verify connection\n");
+        return NULL;
+    }
```

Comment:
Unsure if it is needed.

Reply:
We could just leave it there for debugging and erro analysis purpose.